src/main/java/me/minidigger/hangar/model/viewhelpers/VersionData.java

```diff
@@ -1,20 +1,33 @@
 package me.minidigger.hangar.model.viewhelpers;
 
+import me.minidigger.hangar.db.model.ProjectChannelsTable;
 import me.minidigger.hangar.db.model.ProjectVersionsTable;
-import me.minidigger.hangar.model.generated.Version;
+import me.minidigger.hangar.db.model.ProjectsTable;
+import me.minidigger.hangar.model.Platform;
+import me.minidigger.hangar.model.generated.Dependency;
+import me.minidigger.hangar.util.StringUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 public class VersionData {
 
-    private ProjectData p;
-    private ProjectVersionsTable v;
-//    private Channel c;
-    private String approvedBy;  // Reviewer if present
-    //dependencies: Seq[(Dependency, Option[Model[Project]])]
+    private final ProjectData p;
```

Comment:
These fields should have a more meaningful name

Reply:
issue is that its used all over the templates and shit, IJ doesn't fully catch the refactor...

Reply:
Yeah, if we wanna go back and change template var names and stuff, can do that later, for now. its gonna match Ore.